prov/gni/src/gnix_eq.c

```diff
@@ -93,7 +89,7 @@ static int gnix_verify_eq_attr(struct fi_eq_attr *attr)
 		attr->size = GNIX_EQ_DEFAULT_SIZE;
 
 	/*
-	 * Initial implementation doesn't support any type of wait object.
+	 * We only support FI_WAIT_SET and FI_WAIT_UNSPEC
```

Comment:
Based on discussions from today, continuing to use FI_EINVAL, and no ops table so we can support FI_DIRECT.